### DISCUSSION TITLE
想问一下有没有开源的表单驱动的低代码平台？

### DISCUSSION THREAD

**Discussion by Mengleijin** [taowen/awesome-lowcode#127] [General]
目前开源的好像都是模型驱动的低代码平台啊，有没有开源的表单驱动的，类似简道云、明道云这种

**Comment by ljhskyso** [comment]
> 目前开源的好像都是模型驱动的低代码平台啊，有没有开源的表单驱动的，类似简道云、明道云这种

apitable

**Reply by Mengleijin** [reply]
好的谢谢，我也找到这个了，好像开源的产品都是类似Excel这种形态的

**Comment by vannvan** [comment]
百度的amis算不算？

**Reply by Mengleijin** [reply]
百度的amis应该算前端的低代码，后端接口还是要自己开发的

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬们有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/
